tests/profitbricks/requests/compute/nic_tests.rb

```diff
@@ -44,11 +44,32 @@
 
     tests('#get_all_images').data_matches_schema(@minimal_schema_with_items) do
       getAllImagesResponse = service.get_all_images
-      data = service.get_all_images.body['items'].find { |image|
-        image['properties']['location'] == 'us/las' &&
-        image['properties']['imageType'] == 'CDROM' &&
-        image['properties']['licenceType'] == 'LINUX'
+      
+      data = getAllImagesResponse.body['items'].find { |image|
+        if ENV["FOG_MOCK"] != "true"
+          if image['properties']
+            image['properties']['location'] == 'us/las' &&
+            image['properties']['imageType'] == 'CDROM' &&
+            image['properties']['licenceType'] == 'LINUX'
+          else
+            image['location'] == 'us/las' &&
+            image['imageType'] == 'CDROM' &&
+            image['licenceType'] == 'LINUX'
+          end
+        else
+          if image['properties']
+            image['properties']['location'] == 'us/las' &&
+            image['properties']['imageType'] == 'CDROM' &&
+            image['properties']['licenceType'] == 'UNKNOWN'
+          else
+            image['location'] == 'us/las' &&
+            image['imageType'] == 'CDROM' &&
+            image['licenceType'] == 'UNKNOWN'
+          end
```

Comment:
I do not create an image - see [API documentation](https://devops.profitbricks.com/api/cloud/v2/#images).

Therefor, when live testing, I use a pre-exising image from ProfitBricks. I do not want to change any properties on their pre-existing images, which other people might be using.

In case of mock data, I am able to test the update, because it won't will not affect anyone. And because the mock data was updated/changed in one of the previous tests, it will not be the same as the live data on ProfitBricks servers - which was not modified.